packages/tools/examples/polyDataActorManipulationTools/index.ts

```diff
@@ -101,7 +104,11 @@ function getSphereActor({
  */
 async function run() {
   // Init Cornerstone and related libraries
-  await csRenderInit();
+  await csRenderInit({
+    rendering: {
+      statsOverlay: true,
```

Comment:
i rather have it under debug: { statsOverlay } to indicate it is for debug purpose